content/en/Getting started/glossary.md

```diff
@@ -78,11 +78,15 @@ an individual, a group, or even a nation-state. Specified as "attacker" in Cobal
 
 ## Attestation Letter
 
-{{% attestation-letter %}}
+{{% attestation-letter %}} You cannot customize an Attestation Letter. {{% learn-more-reports %}}
```

Comment:
@mjang-cobalt FYI. There is a plan to make Attestation Letters customizable.

Reply:
I remember reading that, thanks for highlighting.